package.json

```diff
@@ -31,15 +36,26 @@
     "n-common-static-data": "github:Financial-Times/n-common-static-data.git#v1.6.2",
     "n-health": "3.3.1",
     "node-marketo-rest": "^0.7.0",
-    "snyk": "^1.163.3"
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "snyk": "^1.163.3",
+    "sucrase": "^3.10.1"
```

Comment:
We can make `sucrase` a devDependency to keep it with the other testing packages

Reply:
no we can't because we are using it in production ;)